packages/cactus-plugin-ledger-connector-daml/src/test/typescript/integration/daml-get-transaction.test.ts

```diff
@@ -0,0 +1,232 @@
+import { v4 as uuidv4 } from "uuid";
+import "jest-extended";
+
+import { DamlTestLedger } from "@hyperledger/cactus-test-tooling";
+
+import { PluginLedgerConnectorDAML } from "../../../main/typescript/plugin-ledger-connector-daml";
+
+import { DefaultApi } from "../../../main/typescript/generated/openapi/typescript-axios/index";
+
+import {
+  IListenOptions,
+  LogLevelDesc,
+  LoggerProvider,
+  Logger,
+  Servers,
+} from "@hyperledger/cactus-common";
+
+import { pruneDockerAllIfGithubAction } from "../../../../../../packages/cactus-test-tooling/src/main/typescript/github-actions/prune-docker-all-if-github-action";
+import { Configuration } from "@hyperledger/cactus-core-api";
+import http from "http";
+import { AddressInfo } from "net";
+import bodyParser from "body-parser";
+
+import express from "express";
+const logLevel: LogLevelDesc = "TRACE";
+const testLogLevel: LogLevelDesc = "info";
+// Logger setup
+const log: Logger = LoggerProvider.getOrCreate({
+  label: "daml-get-transaction.test",
+  level: testLogLevel,
+});
+
+describe("PluginLedgerConnectorDAML", () => {
+  const damlTestLedger = new DamlTestLedger({
+    imageVersion: "2024-09-08T07-40-07-dev-2cc217b7a",
+    imageName: "ghcr.io/hyperledger/cacti-daml-all-in-one",
+    rpcApiHttpPort: 7575,
+  });
+  let apiClient: DefaultApi;
+  const expressApp = express();
+  const server = http.createServer(expressApp);
+
+  beforeAll(async () => {
+    log.info("Prune Docker...");
+    await pruneDockerAllIfGithubAction({ logLevel: testLogLevel });
+
+    await damlTestLedger.start();
+    expressApp.use(bodyParser.json({ limit: "250mb" }));
+
+    const plugin = new PluginLedgerConnectorDAML({
+      instanceId: uuidv4(),
+      logLevel,
+      apiUrl: "http://localhost:7575/v1",
+    });
+
+    const listenOptions: IListenOptions = {
+      hostname: "127.0.0.1",
+      port: 0,
+      server,
+    };
+
+    const addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
+    const { address, port } = addressInfo;
+    const apiHost = `http://${address}:${port}`;
+    const config = new Configuration({ basePath: apiHost });
+
+    await plugin.registerWebServices(expressApp);
+
+    apiClient = new DefaultApi(config);
+  });
+
+  afterAll(async () => {
+    log.info("FINISHING THE TESTS");
+    if (damlTestLedger) {
+      log.info("Stop the DAML ledger...");
+      await damlTestLedger.stop();
+      await damlTestLedger.destroy();
+    }
+
+    log.info("Prune Docker...");
+    await pruneDockerAllIfGithubAction({ logLevel: testLogLevel });
+  });
+
+  describe("DAML SIMPLE IOU TRANSACTION", () => {
+    it("DAML SIMPLE IOU TRANSACTION", async () => {
```

Comment:
@raynatopedrajeta Please do not use caps lock when writing test descriptions, sentence case is OK for example.